foundation/core/variables.tf

```diff
@@ -121,3 +121,51 @@ variable "enable_point_to_site_vpn" {
   description = "If enabled, creates User Point to Site VPN configuration in CAF foundation."
   default     = true
 }
+
+variable "slack_func_identifier" {
+  type    = string
+  default = "billing-alert-func"
+}
+
+variable "enable_budget_alerts" {
+  type        = bool
+  description = "Feature flag to enable billing alerts"
+  default     = false
+}
+
+variable "func_identifier" {
+  type        = string
+  default     = "billing-alert-func"
+  description = "string used to identity billing alert function resources"
+}
```

Comment:
What is the use case for having this be a variable? Why would we ever want to use something other than `billing-alert-func`. If there isn't a strong use case lets remove the variable and just name the resources accordingly.